salt/states/user.py

```diff
@@ -225,7 +225,7 @@ def _changes(name,
 def present(name,
             uid=None,
             gid=None,
-            gid_from_name=False,
```

Comment:
This is a breaking change. Is there a reason we can't preserve backward compatibility here?

Reply:
Because it doesn't actually do anything now IIRC. Just taking a quick read of https://github.com/saltstack/salt/issues/48640 to jog my memory.... the `gid_from_name` was already broken in a stable point release some months back, and AFAIK the bad commit that broke it was never reverted.

What do you think that argument should now do if we leave it in?